pkg/debugapi/postage.go

```diff
@@ -66,6 +66,14 @@ func (s *Service) postageCreateHandler(w http.ResponseWriter, r *http.Request) {
 		immutable, _ = strconv.ParseBool(val[0])
 	}
 
+	if !s.postageCreateSem.TryAcquire(1) {
+		s.logger.Debug("create batch: resource is busy")
+		jsonhttp.InternalServerError(w, "resource is busy")
+		return
+	} else {
```

Comment:
else is not needed, since you `return` in the if block

Reply:
Done.